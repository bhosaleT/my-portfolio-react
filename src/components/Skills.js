import React from "react";

const Skills = () => (
  <div className="skill-block">
    <div className="skill-block__tab">
      <h4 className="skill-block__heading">
        <img
          className="button-download"
          src={require("../images/svg/code.svg")}
          alt="download button"
        />
        Front End
      </h4>
      <ul className="skill-list">
        <li className="skill-item">HTML5</li>
        <li className="skill-item">CSS3 / Sass</li>
        <li className="skill-item">FlexBox / CSS-GRID</li>
        <li className="skill-item">JavaScript (ES6)</li>
        <li className="skill-item">jQuery</li>
        <li className="skill-item">React</li>
        <li className="skill-item">Redux</li>
      </ul>
    </div>
    <div className="skill-block__tab">
      <h4 className="skill-block__heading">
        <img
          className="button-download"
          src={require("../images/svg/archive.svg")}
          alt="download button"
        />
        Back End
      </h4>
      <ul className="skill-list">
        <li className="skill-item">Node.js</li>
        <li className="skill-item">Express</li>
        <li className="skill-item">Firebase</li>
        <li className="skill-item">Firebase Authentication</li>
      </ul>
    </div>
    <div className="skill-block__tab">
      <h4 className="skill-block__heading">
        <img
          className="button-download"
          src={require("../images/svg/github.svg")}
          alt="download button"
        />
        Other
      </h4>
      <ul className="skill-list">
        <li className="skill-item">Git / Github</li>
        <li className="skill-item">JSON</li>
        <li className="skill-item">Google Maps API</li>
        <li className="skill-item">FourSquare API</li>
        <li className="skill-item">Testing with Jest</li>
        <li className="skill-item">Responsive Design</li>
        <li className="skill-item"> Service Workers </li>
      </ul>
    </div>
  </div>
);

export default Skills;
