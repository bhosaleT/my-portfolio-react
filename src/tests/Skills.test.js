import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Skills from '../components/Skills';

test('The skills component should render without crashing', ()=>{
    const renderer = new ReactShallowRenderer();
    renderer.render(<Skills />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})