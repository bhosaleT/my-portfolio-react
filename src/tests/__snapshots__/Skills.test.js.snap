// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The skills component should render without crashing 1`] = `
<div
  className="skill-block"
>
  <div
    className="skill-block__tab"
  >
    <h4
      className="skill-block__heading"
    >
      <img
        alt="download button"
        className="button-download"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "html-five.svg",
          }
        }
      />
      Front End
    </h4>
    <ul
      className="skill-list"
    >
      <li
        className="skill-item"
      >
        HTML5
      </li>
      <li
        className="skill-item"
      >
        CSS3 / Sass
      </li>
      <li
        className="skill-item"
      >
        FlexBox / CSS-GRID
      </li>
      <li
        className="skill-item"
      >
        JavaScript (ES6)
      </li>
      <li
        className="skill-item"
      >
        jQuery
      </li>
      <li
        className="skill-item"
      >
        React
      </li>
      <li
        className="skill-item"
      >
        Redux
      </li>
    </ul>
  </div>
  <div
    className="skill-block__tab"
  >
    <h4
      className="skill-block__heading"
    >
      <img
        alt="download button"
        className="button-download"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "archive.svg",
          }
        }
      />
      Back End
    </h4>
    <ul
      className="skill-list"
    >
      <li
        className="skill-item"
      >
        Node.js
      </li>
      <li
        className="skill-item"
      >
        Express
      </li>
      <li
        className="skill-item"
      >
        Firebase
      </li>
      <li
        className="skill-item"
      >
        MySQL and NoSQL
      </li>
    </ul>
  </div>
  <div
    className="skill-block__tab"
  >
    <h4
      className="skill-block__heading"
    >
      <img
        alt="download button"
        className="button-download"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "git.svg",
          }
        }
      />
      Other
    </h4>
    <ul
      className="skill-list"
    >
      <li
        className="skill-item"
      >
        Git / Github
      </li>
      <li
        className="skill-item"
      >
        JSON
      </li>
      <li
        className="skill-item"
      >
        Webpack
      </li>
      <li
        className="skill-item"
      >
        Responsive Design
      </li>
      <li
        className="skill-item"
      >
        Jasmine/Jest Testing
      </li>
    </ul>
  </div>
</div>
`;
